sdk/servicebus/azure-servicebus/samples/README.md

```diff
@@ -0,0 +1,48 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-servicebus
+urlFragment: servicebus-samples
+---
+
+# Azure Service Bus client library for Python Samples
+
+These are code samples that show common scenario operations with the Azure Service Bus client library.
+Both [sync version](./sync_sampes) and [async version](./async_samples) of samples are provided, async samples require Python 3.5 or later.
+
+- [topic_send.py](./sync_samples/topic_send.py) ([async version](./async_samples/topic_send_async.py)) - Examples to send messages on a service bus topic:
+    - From a connection string
+    - Enabling Logging
+
+## Prerequisites
+- Python 2.7, 3.5 or later.
+- **Microsoft Azure Subscription:**  To use Azure services, including Azure Service Bus, you'll need a subscription.
+If you do not have an existing Azure account, you may sign up for a free trial or use your MSDN subscriber benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+
+## Setup
+
+1. Install the Azure Service Bus client library for Python with [pip](https://pypi.org/project/pip/):
+```bash
+pip install azure-servicebus
+```
+
+To run samples that utilize the Azure Active Directory for authentication, please install the `azure-identity` library:
+```bash
+pip install azure-identity
```

Comment:
I guess service bus hasn't supported azure-identity yet?

Reply:
Ah, correct, thanks++  (I pulled this README out of EH, trimmed most of the irrelevant sections, but missed this.  The irony being, Adam is currently working to enable it for T2, so we'll be adding this in before long :) )